src/form/Select/Select.tsx

```diff
@@ -119,6 +119,11 @@ export interface SelectProps {
    */
   createable?: boolean;
 
+  /**
+   * The list of KeyCodes for creating select values.
+   */
+  valuesSplitterKeyCodes?: string[];
```

Comment:
Not sure about name

Reply:
A good name for such a property could be `selectOnKeys`. This name is concise, descriptive, and follows the convention of using verb-noun pairs for boolean properties. It clearly communicates that the select action will be triggered on certain key presses.